### DISCUSSION TITLE
Way to set wgpu adapter and performance preference

### DISCUSSION THREAD

**Discussion by VarLad** [JuliaWGPU/WGPUgfx.jl#26] [General]
I can't seem to be able to force WGPUgfx to use my iGPU or low power  mode. 
Is there a way to do it?

**Comment by VarLad** [comment]
@arhik Should I ask in WGPUCore instead?

**Reply by arhik** [reply]
Since I was single developer I was taking short cuts for quick prototyping. I must have not considered it. Will have to check. Can I get back to you later ?

**Reply by arhik** [reply]
I remember a placeholder. Not like legit attempt to choose a GPU.

**Reply by VarLad** [reply]
@arhik Usually it can also be defined with environment variables `WGPU_ADAPTER` and `WGPU_BACKEND`. (doesn't work here)

**Reply by arhik** [reply]
Ahh. No. I was not considering environment variables. Labelling this discussion as enhancement. We need it.
